examples/sampling/graphbolt/node_classification.py

```diff
@@ -311,20 +311,20 @@ def parse_args():
         help="Learning rate for optimization.",
     )
     parser.add_argument(
-        "--batch-size", type=int, default=256, help="Batch size for training."
+        "--batch-size", type=int, default=1024, help="Batch size for training."
```

Comment:
Is there any theory supporting that selecting this value is the best option?

Reply:
Ditto